timesketch/frontend-ng/src/components/LeftPanel/SigmaRules.vue

```diff
@@ -23,11 +23,9 @@ limitations under the License.
       @click="expanded = !expanded"
```

Comment:
**UX feedback:** If the element does not expand to show an "empty state" when there are no sigma rules (which makes sense), then it should not be rendered as clickable. (cursor: pointer).
I would recommend to change line 20 to `:style="!(sigmaRules && sigmaRules.length) ? '' : 'cursor: pointer'"`

Reply:
fixed